pkg/domain/infra/abi/generate.go

```diff
@@ -40,7 +40,7 @@ func (ic *ContainerEngine) GenerateSystemd(ctx context.Context, nameOrID string,
 	// We're either having a pod or garbage.
 	pod, err := ic.Libpod.LookupPod(nameOrID)
 	if err != nil {
-		return nil, err
+		return nil, errors.Errorf("%q does not refer to a container or pod", nameOrID)
```

Comment:
Should be an errors.Wrapf(err, "%q does not refer to a container or pod", nameOrID)

Reply:
Then I will also squeeze in the error from looking up the container from a few lines before.